bitnami/jaeger/values.yaml

```diff
@@ -0,0 +1,882 @@
+  ##
+  extraVolumeMounts: [ ]
+
+  ## Jaeger service parameters
+  ##
+  service:
+    ## @param service.type Jaeger service type
+    ##
+    type: ClusterIP
+    ## @param service.ports.....
+    ## @param service.ports.....
+    ##
+    ports:
+      query_api: 16686
+      query_admin: 16687
+    #    agent-compact: 6831
+    #    agent_binary: 6832
+    #    agent_config: 5778
+    #    agent_zipkin: 5775
+    #    agent_admin: 14271
+    #    collector_grpc: 14250
+    #    collector_binary: 14268
+    #    collector_zipkin: 9411
+    #    collector_admin: 14269
+    ## Node ports to expose
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ## @param service.nodePorts.cql Node port for CQL
+    ## @param service.nodePorts.metrics Node port for metrics
+    ##
+    nodePorts:
+      cql: ""
+      metrics: ""
+    ## @param service.extraPorts Extra ports to expose in the service (normally used with the `sidecar` value)
+    ##
+    extraPorts: [ ]
+    ## @param service.loadBalancerIP LoadBalancerIP if service type is `LoadBalancer`
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+    ##
+    loadBalancerIP: ""
+    ## @param service.loadBalancerSourceRanges Service Load Balancer sources
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ## e.g:
+    ## loadBalancerSourceRanges:
+    ##   - 10.10.10.0/24
+    ##
+    loadBalancerSourceRanges: [ ]
+    ## @param service.clusterIP Service Cluster IP
+    ## e.g.:
+    ## clusterIP: None
+    ##
+    clusterIP: ""
+    ## @param service.externalTrafficPolicy Service external traffic policy
+    ## ref https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+    ##
+    externalTrafficPolicy: Cluster
+    ## @param service.annotations Provide any additional annotations which may be required.
+    ## This can be used to set the LoadBalancer service type to internal only.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+    ##
+    annotations: { }
+    ## @param service.sessionAffinity Session Affinity for Kubernetes service, can be "None" or "ClientIP"
+    ## If "ClientIP", consecutive client requests will be directed to the same Pod
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#virtual-ips-and-service-proxies
+    ##
+    sessionAffinity: None
+    ## @param service.sessionAffinityConfig Additional settings for the sessionAffinity
+    ## sessionAffinityConfig:
+    ##   clientIP:
+    ##     timeoutSeconds: 300
+    ##
+    sessionAffinityConfig: { }
+
+
+
+
+## @section Cassandra storage pod
+##
+cassandra:
+  enabled: true
+
+
+## @section Statefulset parameters
```

Comment:
This section below is not necessary as the settings are set per component. This also looks like Cassandra chart parameters, which we don't want

Reply:
Done [3fda287](https://github.com/bitnami/charts/pull/13480/commits/3fda28743ac0f48a4b011fea95957c8a17234b43)